javascript/ql/src/semmle/javascript/security/dataflow/InsecureDownloadCustomizations.qll

```diff
@@ -58,13 +102,48 @@ module InsecureDownload {
 
   /**
    * A url downloaded by a client-request, seen as a sink for download of
-   * sensitive file through insecure connection.a
+   * sensitive file through insecure connection.
    */
   class ClientRequestURL extends Sink {
     ClientRequest request;
 
     ClientRequestURL() { this = request.getUrl() }
 
     override DataFlow::Node getDownloadCall() { result = request }
+
+    override DataFlow::FlowLabel getALabel() {
+      result instanceof Label::SensitiveInsecureURL // TODO: Also non-sensitive.
```

Comment:
Is this TODO comment still needed? It looks like non-sensitive URLs are handled below
```suggestion
      result instanceof Label::SensitiveInsecureURL
```

Reply:
Correct. I forgot to remove that comment after I implemented that.